types/index.d.ts

```diff
@@ -236,20 +236,16 @@ declare namespace Realm {
 
 
      interface Dictionary{
```

Comment:
Shouldn't the interface declaration be generic since in Core you can constrain the dictionary values to be of specific type?

Reply:
Yes, if they can be constrained, then we should definitely reflect that.

Reply:
@nirinchev Dictionary now allows the value to be constrained